utils/vlog/yosys_json.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/python3
```

Comment:
I wonder if [pyverilog](https://pypi.python.org/pypi/pyverilog) or [hdlparse](https://kevinpt.github.io/hdlparse/) would be useful supplement to yosys here?

I definitely want our verilog models to compatible with [hdlparse](https://kevinpt.github.io/hdlparse/) so we can use [symbolator](https://github.com/kevinpt/symbolator) to generate pretty diagrams like this;
![image](https://user-images.githubusercontent.com/21212/36343250-921845ae-13be-11e8-9534-032457c8710c.png)

Reply:
Not sure, they might be useful for some stuff but I think both are too limited - particularly (like no attribute support), so it seemed easiest to use Yosys for everything. I thing hdlparse at least will ignore the stuff that it doesn't understand though, so we should still get diagrams.